### PR TITLE
Add support for providing Artifactory and JFrog platform URLs

### PR DESCRIPTION
**Pull request by yahavi**
* Add support for providing Artifactory and JFrog platform URLs
* Upgrade build-info dependencies

Precondition to: https://github.com/jfrog/jfrog-idea-plugin/pull/103

### CODE REVIEW

File: src/main/java/com/jfrog/ide/common/utils/Utils.java
Comment (eyalbe4) [inline src/main/java/com/jfrog/ide/common/utils/Utils.java:44]: Any idea how ```fromString``` became ```FromString``` in the build-info code base? :) 
It's a method, not a class. Let's fix.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: src/main/java/com/jfrog/ide/common/utils/Utils.java
Comment (yahavi) [reply src/main/java/com/jfrog/ide/common/utils/Utils.java:44]: It accidently caused by this [commit](https://github.com/jfrog/build-info/commit/fa23cac834a4c12462431f7b177a593986b587b7#diff-427cd6de62dc2aadcc8b7a91fa7adb1ad877517bffd134b6820be815cc4d062aR50) and fixed by this [PR](https://github.com/jfrog/build-info/pull/440).
I'll wait for the next build-info release to keep it `fromString`, before merging this PR.

**Pull request merged**
